packages/design-tokens/properties/theme/fallbacks.json

```diff
@@ -13,6 +13,10 @@
                 "base": {"value": "#14C684"},
                 "inverted": {"value": "#14C684"}
             },
+            "info": {
```

Comment:
FInding I needed to add "info" here was very much non-straight forward for me.

Reply:
Thanks, that's good feedback. Where would you expect to add or edit theme defaults?